### PR TITLE
generation of population of Wroclaw and poland

### PR DESCRIPTION
**Pull request by onomatopeia**

### CODE REVIEW

File: data/simulations/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Simulations
+
+# 20200308_0010
```
Comment (eMaerthin) [inline data/simulations/README.md]: @onomatopeia I don't see this folder :(

File: src/data/preprocessing_poland.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    """
+    Preprocesses the family structure excel for a voivodship from pivoted to melted table for easier further processing.
+    """
+    voivodship = os.path.basename(data_folder)[0]
+    voivodship_folder = os.path.join(data_folder, os.pardir, voivodship)
+    df = pd.read_excel(os.path.join(voivodship_folder, household_family_structure_xlsx.file_name),
+                       sheet_name=household_family_structure_xlsx.sheet_name)
+    df2 = pd.melt(df,
+                  id_vars=['family_type', 'relationship', 'house master', 'family_structure_regex'],
+                  value_vars=[1, 2, 3, 4, 5, 6, 7], var_name='household_headcount',
```
Comment (eMaerthin) [inline src/data/preprocessing_poland.py]: here we hardcode to `[1, 2, 3, 4, 5, 6, 7]` - is this expected in general to have headcount up to 7?

**Comment by onomatopeia** [comment]
Proprocessing code is highly coupled with the format of underlying excel
files. In this particular case, these are the columns i'm explicitly
interested in.

**Comment by onomatopeia** [comment]
My enums were there first :) Just went unnoticed ;)

File: data/simulations/README.md (hunk 1 above)
Comment (onomatopeia) [reply data/simulations/README.md]: Fixed

**Comment by onomatopeia** [comment]
@eMaerthin Pity you didn't mark these code repetitions, as I don't see them this way, but will keep my eyes open.

**Pull request merged**